src/components/AppVersions/AppVersions.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import styles from "./styles";
+
+interface AppVersionsProps {
+  dashboardVersion: string;
+  coreVersion: string;
+}
+
+const AppVersions: React.FC<AppVersionsProps> = ({
+  dashboardVersion,
+  coreVersion
+}) => {
+  const classes = styles({});
+
+  return (
+    <div className={classes.container}>
+      <div
```

Comment:
Why not to use `span`, `p` or `Typography` component?

Reply:
I changed to Typography  component.